src/test/unit_tests/test_bucketspace.js

```diff
@@ -478,9 +472,10 @@ mocha.describe('bucket operations - namespace_fs', async function() {
         assert.ok(list_ok);
     });
     mocha.it('list buckets with dn', async function() {
+        // Give s3_correct_uid access to the required buckets
         const res = await s3_correct_dn_default_nsr.listBuckets({});
         console.log(inspect(res));
-        const list_ok = bucket_in_list([bucket_name + '-s3-2'], [], res.Buckets);
+        const list_ok = bucket_in_list([bucket_name + "-s3-2"], [], res.Buckets);
```

Comment:
Why did you change it to double qoutes?

Reply:
it came back during conflicts resolutions